app/Livewire/Team/InviteLink.php

```
@@ -45,9 +45,16 @@ private function generateInviteLink(bool $sendEmail = false)
         try {
             $this->authorize('manageInvitations', currentTeam());
             $this->validate();
-            if (auth()->user()->role() === 'admin' && $this->role === 'owner') {
+
+            // Prevent privilege escalation: users cannot invite someone with higher privileges
+            $userRole = auth()->user()->role();
+            if ($userRole === 'member' && in_array($this->role, ['admin', 'owner'])) {
+                throw new \Exception('Members cannot invite admins or owners.');
```

PHPMD (2.15.0)

Missing class import via use statement (line '52', column '27'). (undefined)

(MissingImport)